llvm/include/llvm/Transforms/Instrumentation/AddressSanitizer.h

```diff
@@ -20,12 +20,20 @@ namespace llvm {
 class Module;
 class raw_ostream;
 
+static const int InstrumentationWithCallsThreshold = 7000;
```

Comment:
Please move them into 

AddressSanitizerOptions

they do not deserve to be in llvm:: top namespace